packages/fluentui/accessibility/src/behaviors/Datepicker/datepickerCalendarCellBehavior.ts

```diff
@@ -5,14 +5,19 @@ import { Accessibility } from '../../types';
  * Behavior for a datepicker calendar grid.
```

Comment:
Please could you add justification why we are adding role "none" and "gridcell" to the respective slots?